assets/src/scripts/monitoring-location/components/hydrograph/selectors/parameter-data.js

```diff
@@ -25,7 +25,7 @@ export const getAvailableParameterCodes = createSelector(
         const seriesList = Object.values(timeSeries);
         const availableVariableIds = seriesList.map(x => x.variable);
         return sortedParameters(variables)
-            .filter(variable => availableVariableIds.includes(variable.oid))
+            .filter(variable => availableVariableIds.includes(variable.oid.split(`_${config.CALCULATED_TEMPERATURE_VARIABLE_CODE}`)[0]))
```

Comment:
This is confusing to me why you need to do this. Also after running the code and looking at the state in ivTimeSeriesData.variables, the calculated variable is this:
```
variableCode: {
 (the other properties)
variableID: 45807042F
}
```
I think I would prefer for the variableID to be 45807042F (without underscore) everywhere (so for the key values too). Maybe I'm missing something but this seems very confusing to me.

Reply:
lol. This is a hold over from a previous idea. But fixed now.